src/main/kotlin/com/doist/detekt/NoNotNullOperator.kt

```diff
@@ -0,0 +1,32 @@
+package com.doist.detekt
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+import org.jetbrains.kotlin.psi.KtPostfixExpression
+
+class NoNotNullOperator(config: Config) : Rule(config) {
+    override val issue = Issue(
+        javaClass.simpleName,
+        Severity.Style,
+        "Detects !! operator",
+        Debt.FIVE_MINS,
+    )
+
+    override fun visitPostfixExpression(expression: KtPostfixExpression) {
+        super.visitPostfixExpression(expression)
+
+        if (expression.text.endsWith("!!")) {
```

Comment:
@panpeter was checking this PR after you linked it on your personal DO thread and just wanted to ask: shouldn't this be a `contains` instead of `endsWith`? Otherwise we can have something like `val size = nullableList!!.size` and the rule wouldn't flag this as an issue, but the issue is still there. Please disregard if it's covered differentely, it was just a thought after skimming through the PR. 😊

Reply:
Thanks for checking this code 😊

Good question! I'm also new to all these [PSI element names](https://plugins.jetbrains.com/docs/intellij/psi-elements.html) and I wasn't sure so I've added [a test](https://github.com/Doist/detekt-rules/commit/a5528254f66f47090c62b63a98ea7f49f39537a8). Fortunately, the rule covers this case.

The method we use in this rule is `visitPostfixExpression` so in your example the expression is just the first part `nullabeList!!` without the `.rule` 👍